### ISSUE TITLE
WCAG Aria / Role Attributes

### ISSUE THREAD

**Issue by drolsen** [brianvoe/slim-select#244]
With the ever growing specs around aria attributes used to better describe abstracted markup structures to screen readers, the markup generated by slim select should really be distributing some aria attributes now.

https://www.w3.org/TR/wai-aria-practices-1.1/examples/listbox/listbox-collapsible.html

Three important changes:
- The .ss-list element should have `role="listbox"` and .ss-option elements should have `role="option"`.
- The .ss-list element should utilize the `aria-labelledby="XXXX"` attribute which should have a id relationship with the .placeholder span.
- Original native select should be given a `aria-hidden="true"` so that screen readers don't try to travers both native and slim select.

There are other requirements around using the `aria-selected="true"` on selected .ss-option elements for screen readers, but you can consider that bonus points if you will.

Thanks for listening and creating slim select!

**Comment by brianvoe** [comment]
Thanks for the kind words. Im more than willing to merge any good pr's that you would like to submit. Please submit something and we can get that merged in asap.

**Comment by claudiu-cristea** [comment]
@drolsen is this what we need? https://github.com/claudiu-cristea/slim-select/compare/b10531db42029c9758fbe8fcfb7260a2816d3038...claudiu-cristea:3130a27df2be84c2eff927d7db308ffc84554fbc
